website/docs/cloud-docs/api-docs/changelog.mdx

```diff
@@ -9,6 +9,10 @@ description: >-
 
 Keep track of changes to the API for Terraform Cloud and Terraform Enterprise.
 
+## 2023-09-08
+* Updated the [Organization API](/terraform/cloud-docs/api-docs/organizations) to include `default-execution-mode` and `default-agent-pool`.
+* Updated the [Workspaces API](/terraform/cloud-docs/api-docs/workspaces) to include the new `setting-overwrites` object to allow for overwriting the new organization defaults.
```

Comment:
Does this affect all defaults for an org or just  `default-execution-mode` and `default-agent-pool`? If it's all defaults I suggest the below:
```suggestion
* Update the [Workspaces API](/terraform/cloud-docs/api-docs/workspaces) to add a `setting-overwrites` object to allow for overwriting an organization's defaults.
```

If it's just  `default-execution-mode` and `default-agent-pool` I suggest this:

```suggestion
* Update the [Workspaces API](/terraform/cloud-docs/api-docs/workspaces) to add a `setting-overwrites` object to allow you to overwrite `default-execution-mode` and `default-agent-pool`.
```